msresist/figures/figureS2.py

```diff
@@ -0,0 +1,29 @@
+"""
+This creates Supplemental Figure 2: Cell M time course
+"""
+
```

Comment:
Same as with S1. Also, many journals limit the number of supplementary figures now to either supplementary figures must correspond to one of the main figures, or there can be no greater number than main figures. Should each phenotype have its own supplementary figure? Are you going to reference these plots in the text?